utils/docker/Dockerfile.el.9

```diff
@@ -0,0 +1,206 @@
+# Copyright 2022 Intel Corporation
+# All rights reserved.
+#
+# 'recipe' for Docker to build an image of EL 9 based
+# environment for building the DAOS project.
+#
+
+# Pull base image
+ARG BASE_DISTRO=almalinux:9
+FROM $BASE_DISTRO
+LABEL maintainer="daos@daos.groups.io"
+
+# Intermittent cache-bust.  Used to reduce load on the actual CACHEBUST later.
+ARG CB0
+
+# Use local repo server if present
+ARG REPO_FILE_URL
+RUN set -e;                                                      \
+    if [ -n "$REPO_FILE_URL" ]; then                             \
+        cd /etc/yum.repos.d/ &&                                  \
+        curl -k -f -o daos_ci-el9-artifactory.repo.tmp           \
+             "$REPO_FILE_URL"daos_ci-el9-artifactory.repo &&     \
```

Comment:
This won't work as https://github.com/daos-do/repo-files/ doesn't have any EL9 repo files defined yet.  That will need to be done before we can do any EL9 testing in Jenkins.

Reply:
Understood, just wasn't sure what to put there in the meantime.

Reply:
Indeed.  What you put is exactly right.  The file is just not there yet.  And quite honestly, this could land before that repo file is even created as this PR is not actually adding any building or testing of EL9 in Jenkins yet.